src/main/java/example/models/PlayerStats.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Based on how we setup the unit tests in elide-spring-boot-auto-confiugure, we dont need these Java classes.

Reply:
we can just create the table and then just have hjsons against them.